core/src/main/java/com/linecorp/armeria/server/AbstractHttpResponseHandler.java

```diff
@@ -183,7 +183,7 @@ final AggregatedHttpResponse toAggregatedHttpResponse(HttpStatusException cause)
         final Throwable cause0 = firstNonNull(cause.getCause(), cause);
         final ServiceConfig serviceConfig = reqCtx.config();
         final AggregatedHttpResponse response =
-                serviceConfig.server().config().errorHandler()
+                serviceConfig.server().config().errorHandler().asServiceErrorHandler()
                              .renderStatus(serviceConfig, req.headers(), status, null, cause0);
```

Comment:
The current implementation may not be correct if the user specified his/her own `ServiceErrorHandler`

```suggestion
                serviceConfig.errorHandler().renderStatus(serviceConfig, req.headers(), status, null, cause0);
```

Reply:
That's right.
I fixed, thank you~